Makefile.PL

```diff
@@ -12,21 +12,17 @@ MAKE_GOTOOLS
 WriteMakefile(
     NAME      => 'Percona::Toolkit',
     VERSION   => '3.7.0-2',
-    EXE_FILES => [
-      map {
-         (my $name = $_) =~ s/^bin.//;
-         my $file_name = $_;
-         if ( ( $file_name !~ m/mongo/ ) || ( $file_name !~ m/pg/ ) || ( $file_name !~ m/pt-stalk/ )  || ( $file_name !~ m/pt-k8s/ )  ) {
-	     $_;
-         }
-      } <bin/*>
-    ],
+    EXE_FILES => [ <bin/*> ],
     MAN1PODS  => {
       'docs/percona-toolkit.pod' => 'blib/man1/percona-toolkit.1p',
       map {
          (my $name = $_) =~ s/^bin.//;
          my $file_name = $_;
-         if ( ( $file_name !~ m/mongo/ ) || ( $file_name !~ m/pg/ ) || ( $file_name !~ m/pt-stalk/ )  || ( $file_name !~ m/pt-k8s/ )  ) {
+         if ( ( $file_name =~ m/mongo/ ) || ( $file_name =~ m/pg/ ) || ( $file_name =~ m/galera/ )  || ( $file_name =~ m/k8s/ ) || ( $file_name =~ m/secure/ ) ) {
```

Comment:
maybe it will looks better 
```suggestion
if ( $file_name =~ m/(mongo|pg|galera|k8s|secure)/ ) {
```